relay-server/src/actors/global_config.rs

```diff
@@ -26,6 +26,7 @@ pub enum GlobalConfiguration {
 
 impl Interface for GlobalConfiguration {}
 
+/// The message for requesting the most recent global config from [`GlobalConfigurationService`]
```

Comment:
Other reviewers do this to me all the time, now it's my turn 😉 

```suggestion
/// The message for requesting the most recent global config from [`GlobalConfigurationService`].
```